### DISCUSSION TITLE
Can you provide an official document if possible?

### DISCUSSION THREAD

**Discussion by percyzt** [unicape/use-wagmi#35] [Q&A]
Hello, without an official document of our own, it may make us unclear which areas are available, and we are worried that they may not be synchronized with Wagmi's features. Can you use a document similar to GitBook to provide us with one?

**Comment by unicape** [comment]
> Hello, without an official document of our own, it may make us unclear which areas are available, and we are worried that they may not be synchronized with Wagmi's features. Can you use a document similar to GitBook to provide us with one?

Sorry I don't have enough time to do this at the moment I would be very grateful if someone could provide some help Currently the API is basically the same and there is no difference

**Comment by unicape** [comment]
@youthinn In fact, I am doing this outside of work, and now I am preparing to implement a Vue version of rainbowkit and preparing for the update of wagmi v2, so it is really difficult to have time, but I will ensure that there is a complete document in the future

**Reply by percyzt** [reply]
Okay, I hope there will be more open-source developers like you, so that our entire ecological environment can be more perfect!!!

**Reply by reslear** [reply]
@unicape 🚀 have you also considered systems such as https://docs.family.co/connectkit ?

**Reply by unicape** [reply]
no

**Comment by ennioVisco** [comment]
@unicape I might work on a fork of the wagmi docs for this, would you be interested?

**Reply by unicape** [reply]
There are currently no plans to develop use-wagmi documentation because there is no income and maintenance services require costs.
